homework-submission/week1-project/src/Todo app/TodoApp.js

```diff
@@ -0,0 +1,26 @@
+import {observable, computed, action} from 'mobx';
```

Comment:
You missed This 👌
```suggestion
import {observable, computed, action, configure} from 'mobx';
configure({ enforceActions: 'observed' });
```

You don't have to put it there but for a development purposes it's preferred to be here always while doing it.